### PR TITLE
Update global nav

### PR DESCRIPTION
**Pull request by carkod**
## Done

- Update global nav
- Dotrun github action fixed
- Using `versioned_static` for all scripts.

## QA
- Check that https://multipass-run-316.demos.haus/ shows the latest version of global nav
- Top navigation bar is removed and Github links have been moved next to the CTA
image
 
- Check https://multipass-run-316.demos.haus/install tabs work correctly
- If you are using MacOS would be nice also to check that the second tab is selected.

## Issues

Fixes https://github.com/canonical/multipass.run/issues/309
Closes https://warthogs.atlassian.net/browse/WD-7014

### CODE REVIEW

File: .stylelintrc [hunk 1]
```diff
@@ -5,7 +5,7 @@
   ],
   "ignoreFiles": ["static/sass/_hljs.scss"],
   "rules": {
-    "order/properties-alphabetical-order": true,
+    "order/properties-alphabetical-order": null,
```
Comment (petesfrench) [inline .stylelintrc:8]: Could we not remove this rule entirely and subsequently the 'stylelint-order' plugin? It seems like it isn't being used

File: templates/_layouts/default.html [hunk 2]
```diff
@@ -112,12 +109,14 @@
...
-  <script src="/static/js/modules/cookie-policy.js"></script>
-  <script src="/static/js/tabs.js"></script>
+  <script src="{{ versioned_static('js/modules/global-nav.js') }}"></script>
+  <script src="{{ versioned_static('js/modules/cookie-policy.js')}}"></script>
+  <script src="{{ versioned_static('js/tabs.js') }}"></script>
   <script>
     cpNs.cookiePolicy();
-    canonicalGlobalNav.createNav({maxWidth:"72rem"});
+    if (typeof canonicalGlobalNav !== "undefined") {
+      canonicalGlobalNav.createNav({breakpoint: 940});
```
Comment (petesfrench) [inline templates/_layouts/default.html:118]:
```suggestion
      canonicalGlobalNav.createNav({breakpoint: 1035});
```
This needs to be the same as the navigation breakpoint threshold or there will be some visual glitches

**Review by petesfrench — APPROVED** [review]
Could minor changes, but LGTM. There also an alignment issue that will be fixed by upgrading vanilla. But not sure you want to tackle that in this pr.

File: templates/_layouts/default.html (hunk 2 above)
Comment (carkod) [reply templates/_layouts/default.html]: Thanks, updated.

**Pull request merged**